test/snap-driver.js

```diff
@@ -1,8 +1,16 @@
-/* globals SpriteMorph, SnapActions, localize, Point */
 function SnapDriver(world) {
     this._world = world;
+    this._window = window;
 }
 
+SnapDriver.prototype.setWindow = function(window) {
+    this._window = window;
+};
+
+SnapDriver.prototype.globals = function() {
```

Comment:
the `class` syntax provides getters which kinda make the getter methods a little nicer. 
`getter globals() {..}` then you can call it `driver.globals` 
But it also makes distinguishing between attributes and methods more difficult.